tests/Resources/Page/ODH/JupyterHub/PVCChange.resource

```diff
@@ -0,0 +1,71 @@
+*** Settings ***
+Documentation   Set of Keywords for PVC change support
+
+
+*** Keywords ***
+Check If PVC Change Is Permanent
+   [Documentation]   Check if PVC changes are not reverted back
+   [Arguments]    ${NUMBERS}
+   May Be Delete PVC
+   May Be Delete Notebook POD    rhods-notebooks
+   ${size}    Get PVC Size    redhat-ods-applications
+   Change PVC Size From ConfigMap     ${NUMBERS}
+   Sleep    15s
+   ${new_size}    Get PVC Size    redhat-ods-applications
+   IF  ${new_size} == ${NUMBERS}[:-2]
+       Should Not Be Equal As Integers    ${size}    ${new_size}
+   ELSE
+       Fail    Unable to change the default PVC size
+   END
+
+Verify Notebook Size
+  [Arguments]    ${timeout}     ${size}
+  [Documentation]   Launch JH spawner and compare size
+  Launch JupyterHub Spawner From Dashboard
+  Spawn Notebook With Arguments    image=s2i-minimal-notebook    spawner_timeout=${timeout}
+  Run Cell And Check Output    ${SIZE_CODE}   ${size}
+  Fix Spawner Status
+
+Verify PVC Size
+   [Documentation]   Compare PVC size
+   [Arguments]    ${actual_size}     ${expected_size}
+   Should Be Equal As Integers    ${actual_size}     ${expected_size}
+
+PVC Size Test Teardown
+   [Documentation]    PVC change teardown
+   SeleniumLibrary.Close All Browsers
+   May Be Delete PVC
+   May Be Delete Notebook POD    rhods-notebooks
+   Change PVC Size From ConfigMap     20Gi
+   Roll Out Jupyter Deployment Config
+
+Roll Out Jupyter Deployment Config
+   [Documentation]    Rollout deployment config for JH
+   ${rollout_result}    Run Keyword And Return Status
+   ...    Run    oc rollout latest dc/jupyterhub -n ${NAMESPACE}
+   Should Be Equal    "${rollout_result}"    "True"
+   Sleep    120s
+
+May Be Delete PVC
+   [Documentation]    Delete PVC from openshift
+   ${status}    ${pvc_name}    Run Keyword And Ignore Error
+   ...     Get User Notebook PVC Name    ${TEST_USER.USERNAME}
+   Run Keyword And Return Status    OpenShiftCLI.Delete    kind=PersistentVolumeClaim
+   ...   field_selector=metadata.name==${pvc_name}    namespace=rhods-notebooks
+
```

Comment:
Perhaps the user should be an argument?

Reply:
i don't see value in it since i am use the get notebook name here

Reply:
Then why do you need the user as an argument to `Get Notebook PVC Size`? I think it'd be better to have it as an argument for all three keywords so that you can change and verify the sizes for different users rather than only the one you are currently logged in as (e.g. if you were testing this as an admin and changing the size for one specific non-admin user)

Reply:
Sure i could do that not a prob

Reply:
done